doc/method/introduction.md

```diff
@@ -11,13 +11,12 @@ Life project, an aggregation of phylogenetic trees with tools for
 operating on them.
 
 In order to meet Open Tree's project requirements, the taxonomy is an automated
-synthesis of ten different source taxonomies with different strengths. The
-synthesis process is repeatable so that updates to source taxonomies can be
-incorporated easily.
+synthesis of ten different source taxonomies. The
+synthesis process is repeatable so that we can easily incorporate updates to source taxonomies.
```

Comment:
a 2nd big reason for repeatability is so that we can easily try out potential improvements to the assembly method.